Corona-Warn-App/src/main/res/layout/contact_diary_overview_nested_list_item.xml

```diff
@@ -34,21 +34,23 @@
         android:ellipsize="end"
         android:focusable="true"
         android:maxLines="3"
-        app:layout_constraintBottom_toBottomOf="@+id/contact_diary_overview_element_image"
+        app:layout_constraintBottom_toTopOf="@+id/contact_diary_overview_element_attributes"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toEndOf="@+id/contact_diary_overview_element_image"
         app:layout_constraintTop_toTopOf="@+id/contact_diary_overview_element_image"
         tools:text="Andrea Steinhauer" />
 
     <TextView
+        tools:visibility="gone"
         android:id="@+id/contact_diary_overview_element_attributes"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:maxLines="3"
-        android:text="unter 15 Min, im Freien"
+        app:layout_constraintBottom_toBottomOf="@+id/contact_diary_overview_element_image"
```

Comment:
Many thanks for the fix! I suppose this bottom constraint will break the layout with multiple lines of notes, does it? 
Otherwise, the PR looks good to me. Could you please target the `release/2.0.x` branch instead of `main`?

Reply:
Good catch! It does actually work, because of `layout_height="wrap_content"`.
![image](https://user-images.githubusercontent.com/32238636/112185090-327eee00-8c00-11eb-9512-1912bc7f39e3.png)